src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/IVsEnvironmentServices.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using Microsoft.CodeAnalysis;
+using System.Threading.Tasks;
+
+namespace Microsoft.VisualStudio.ProjectSystem.VS
+{
+    internal interface IVsEnvironmentServices
```

Comment:
Can we change this a little to:

1) Not include start with IVs - we reserve these for types that actually come from VS.
2) Be a little more isolated, or more specific to rename. This looks like a grab bag of VS functionality on things you just wanted to isolate. For example, display errors to the user to completely different concept to applying changes to Roslyn. They shouldn't be tied to each other.